docs/fabric-renderer.md

```diff
@@ -3,26 +3,26 @@ id: fabric-renderer
 title: Fabric Rendering
 ---
 
-Fabric is React Native's new rendering system. It is designed to improve interoperability with mobile platforms and to unlock new capabilities. Fabric is a conceptual evolution of the render system currently used by React Native. The core principle is to unify more render logic in C++ versus spread across the different [host platforms](glossary#host-platform).
+Fabric is React Native's new rendering system. It was designed, built, and rolled out at Meta between 2018 and 2021. It is conceptual evolution of the legacy render system. The core principles are to unify more render logic in C++, improve interoperability with [host platforms](glossary#host-platform), and to unlock new capabilities.
```

Comment:
Some word-smithing suggestions:


Fabric is React Native's new rendering system, a conceptual evolution of the legacy render system. The core principles are to unify more render logic in C++, improve interoperability with [host platforms](glossary#host-platform), and to unlock new capabilities for React Native. In 2021, it has been adopted by the majority of React Native surfaces in Meta.